packages/utils/src/utils/dates/timeAgo.ts

```diff
@@ -0,0 +1,26 @@
+const periodValues: Record<string, number> = {
+  year: 31536000,
+  month: 2592000,
+  week: 604800,
+  day: 86400,
+  hour: 3600,
+  min: 60,
+};
+
+export const getTimeAgo = (timeValue: string | number) => {
```

Comment:
This is a pretty small function. Let's add it to the `dateFormatters.ts` file.

Reply:
done